package.json

```diff
@@ -67,6 +67,8 @@
     },
     "files": [
         "dist",
-        "*.md"
+        "*.md",
+        "oas30.d.ts",
+        "oas31.d.ts",
```

Comment:
Please, remove the trailing comma on line 72.

Reply:
Sorry my bad, was having lunch when I pushed it. 🙏